packages/core/src/components/Form.js

```diff
@@ -68,8 +68,8 @@ export default class Form extends Component {
     let { errors, errorSchema } = mustValidate
       ? this.validate(formData, schema, additionalMetaSchemas, customFormats)
       : {
-          errors: state.errors || [],
-          errorSchema: state.errorSchema || {},
+          errors: (!props.noValidate && state.errors) || [],
```

Comment:
Could you clean up the code that's here a bit? I think over time & many commits it's become difficult to read. Something like

```
let errors = !props.noValidate ? state.errors : [];
```

Reply:
I moved the logic for getting state errors to a separate function.